doc/api/stream.md

```diff
@@ -568,6 +568,24 @@ added: v12.3.0
 
 Getter for the property `objectMode` of a given `Writable` stream.
 
+##### `writable.updateWritableHighwaterMark(highwaterMark)`
```

Comment:
I can actually see a case for this part and updating the watermarks. I even needed to use the lowWatermark at some point for a specific use case at my old job and was sad to find out we don't have one anymore :] 

I think this sort of change would need to come with a concrete use case of why I'd update the watermarks.